finance/ledger/Portfile

```diff
@@ -42,9 +48,10 @@ configure.args-append \
                     -DGMP_PATH=${prefix}/include \
                     -DMPFR_LIB=${prefix}/lib/libmpfr.dylib \
                     -DMPFR_PATH=${prefix}/include \
-                    -DPYTHON_EXECUTABLE=${prefix}/bin/python2.7 \
-                    -DUSE_PYTHON27_COMPONENT=ON \
+                    -DPYTHON_EXECUTABLE=${prefix}/bin/python3.7 \
+                    -DUSE_PYTHON38_COMPONENT=ON \
                     -DUSE_DOXYGEN=OFF \
                     -DUSE_PYTHON=ON
 
+configure.python ${prefix}/bin/python3.8
```

Comment:
Does configure.python (which sets the PYTHON environment variable) actually do anything for this build system? I'm guessing it does not, since it's specified with -DPYTHON_EXECUTABLE above. In that case, you can remove this line.